Otherwise the option --force-rpath won't work.

--- a/patchelf.cpp
+++ b/patchelf.cpp
@@ -1176,8 +1176,10 @@
         dynRPath = 0;
     }
 
-    if (forceRPath && dynRPath && dynRunPath) { /* convert DT_RUNPATH to DT_RPATH */
-        dynRunPath->d_tag = DT_IGNORE;
+    if (forceRPath && !dynRPath && dynRunPath) { /* convert DT_RUNPATH to DT_RPATH */
+        dynRunPath->d_tag = DT_RPATH;
+        dynRPath = dynRunPath;
+        dynRunPath = 0;
     }
 
     if (newRPath.size() <= rpathSize) {
