--- a/patchelf.cpp
+++ b/patchelf.cpp
@@ -1497,8 +1497,8 @@
   [--set-interpreter FILENAME]\n\
   [--page-size SIZE]\n\
   [--print-interpreter]\n\
-  [--print-soname]\t\tPrints 'DT_SONAME' entry of .dynamic section. Raises an error if DT_SONAME doesn't exist\n\
-  [--set-soname SONAME]\t\tSets 'DT_SONAME' entry to SONAME.\n\
+  [--print-soname]\n\
+  [--set-soname SONAME]\n\
   [--set-rpath RPATH]\n\
   [--remove-rpath]\n\
   [--shrink-rpath]\n\
